keylime-ansible-gcp/roles/create-gcp/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
If we don't have any defaults, then we can just omit this file. It's not necessary to have a blank file/directory in the repo and if anyone ever needs it in the future they can always add it.